crates/typst/src/foundations/int.rs

```diff
@@ -222,6 +263,23 @@ impl Repr for i64 {
     }
 }
 
+/// The endianness of a bytes to and from int conversion.
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Hash)]
+enum Endianness {
```

Comment:
I am not sure if we should make this type public.

Reply:
I guess it could be, but I am waiting on response if we are also implementing bytes to float conversion. That way it, this enum would be better placed in the Bytes file.

Reply:
should be public